app/models/address.rb

```diff
@@ -4,6 +4,7 @@ class Address < ApplicationRecord
   STATE_TO_LOCALE = {
     punjab: 'pa-Guru-IN',
     maharashtra: 'mr-IN',
+    karnataka: 'kn-IN'
```

Comment:
Iiinteresting, so the language chosen for the reminder is not based on the patient's registration facility, but based on their address's "state"?

Reply:
Yeah, we took this decision [here](https://docs.google.com/document/d/1r0lfpK6--PlJEZbnGVx1-KnnMwMBfCk5jErpnJopZzY/edit#heading=h.com9rkcwqvda). But I can't remember exactly why. Best guess would be that we uncovered that patients move around and their specified address is most reliable.

This is not very multi-country sensitive though since it sits in the model. But it can be if we later migrate it to a config file or something. Should not be very hard.

Reply:
Neato 👍 that clarifies why we need to update this map, and why you were [looking at state distribution](https://simpledotorg.slack.com/archives/CFHMC60P5/p1588240632089300) 😄